pinc/project_quick_check.inc

```diff
@@ -191,6 +191,8 @@ function _test_project_for_large_page_images($projectid)
 
 function _test_project_for_corrupt_pngs($projectid)
 {
+    global $page_image_size_limit;
```

Comment:
Doesn't look like this is used in this function?

Reply:
@chrismiceli - this change looks unnecessary.

Reply:
Sorry I forgot to address this and somehow missed it when reviewing. Taken care of now.